src/script/stack.zig

```diff
@@ -158,6 +158,30 @@ pub const Stack = struct {
         return std.mem.readVarInt(i64, elem, native_endian);
     }
 
+    /// Removes an item from the stack at the specified index
+    ///
+    /// # Arguments
+    /// - `index`: The index of the item to remove (0 is the top of the stack)
+    ///
+    /// # Returns
+    /// - `StackError` if the index is out of bounds
+    /// - `[]u8`: The removed item
+    pub fn removeAt(self: *Stack, index: usize) StackError![]u8 {
```

Comment:
I think you are rewriting https://ziglang.org/documentation/master/std/#std.array_list.ArrayListAligned.orderedRemove